test/test-addon.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+var test = require('tape')
+var execFile = require('child_process').execFile
+var path = require('path')
+var addonPath = path.resolve(__dirname, 'node_modules', 'hello_world')
+var nodeGyp = path.resolve(__dirname, '..', 'bin', 'node-gyp.js')
+
+test('build simple addon', function (t) {
+  t.plan(3)
+
+  // Set the loglevel otherwise the output disappears when run via 'npm test'
+  var proc = execFile(process.execPath, [nodeGyp, 'rebuild', '-C', addonPath, '--loglevel=verbose'], function (err, stdout, stderr) {
```

Comment:
Can you try keeping this line < 80 columns, e.g., by declaring the args aray as a variable first?